src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.html

```diff
@@ -111,6 +112,14 @@
           [style.visibility]="(!isCoherent(node) || hasIncoherentDependencies(node)) ? 'visible' : 'hidden'"></fa-icon>
         {{node.build.azureDevOpsBuildNumber}}
       </td>
+      <td>
+        <span *ngIf="node.build.staleness == '0'">
+          latest
+        </span>
+        <span *ngIf="node.build.staleness != '0'">
+          {{node.build.staleness}} behind
```

Comment:
Might be good to define what we are behind on here (ie, builds behind, rather than just behind).